src/bin/lpython.cpp

```diff
@@ -882,12 +885,47 @@ int compile_python_to_object_file(
         print_time_report(times, time_report);
         return 3;
     }
-    m = std::move(res.result);
-    auto llvm_start = std::chrono::high_resolution_clock::now();
-    e.save_object_file(*(m->m_m), outfile);
-    auto llvm_end = std::chrono::high_resolution_clock::now();
-    times.push_back(std::make_pair("LLVM to binary", std::chrono::duration<double, std::milli>(llvm_end - llvm_start).count()));
-    print_time_report(times, time_report);
+    std::unique_ptr<LCompilers::LLVMModule> m = std::move(res.result);
+
+    if (to_jit) {
+        LCompilers::LPython::DynamicLibrary cpython_lib;
+        LCompilers::LPython::DynamicLibrary symengine_lib;
+
+        if (compiler_options.enable_cpython)
+            LCompilers::LPython::open_cpython_library(cpython_lib);
+        if (compiler_options.enable_symengine)
+            LCompilers::LPython::open_symengine_library(symengine_lib);
```

Comment:
I think we should use braces so that it is consistent with the rest of the code base.
```suggestion
        if (compiler_options.enable_cpython) {
            LCompilers::LPython::open_cpython_library(cpython_lib);
        }
        if (compiler_options.enable_symengine) {
            LCompilers::LPython::open_symengine_library(symengine_lib);
        }
```

Reply:
We should update this at every place.